secrets_test.go

```diff
@@ -0,0 +1,95 @@
+package opslevel_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/opslevel/opslevel-go/v2023"
+	"github.com/rocktavious/autopilot/v2023"
+)
+
+func TestCreateSecret(t *testing.T) {
+	// Arrange
+	request := `{
+  "query": "mutation SecretsVaultsSecretCreate($alias:String!$input:SecretInput!){secretsVaultsSecretCreate(alias: $alias, input: $input){secret{alias,id,owner{alias,id},timestamps{createdAt,updatedAt}},errors{message,path}}}",
+  "variables": {{ template "secret_create_vars" }}
+  }`
+	response := `{{ template "secret_create_response" }}`
+	id2 := opslevel.NewID("Z2lkOi8vOTg3NjU0MzIxMTIzNDU2Nzg5")
+	client := ABetterTestClient(t, "secrets/create", request, response)
+	fmt.Println(client)
+	// Act
+	secretInput := opslevel.SecretInput{
+		Owner: opslevel.IdentifierInput{Id: *id2},
```

Comment:
Could also use
```suggestion
		Owner: opslevel.NewIdentifier("Z2lkOi8vOTg3NjU0MzIxMTIzNDU2Nzg5"),
```
but i see why you didn.  Just letting you know we have helpers for safely creating IdentifierInput too